components/clientComponents/forms/AddressComplete/utils.ts

```diff
@@ -150,7 +150,7 @@ export const getAddressAsAnswerElements = (
 
 // Helper function to test if the address has multiple results.
 export function matchesAddressPattern(input: string): boolean {
-  // See issue # 4417  
-  const pattern = /^.+,\s+[A-Z]{2}(?:,\s+[A-Z0-9]+)?\s+-\s+\d+\s+Addresses$/;
+  // Updated regex pattern to make the comma after the city name optional
```

Comment:
We should keep a ref to the PR number and or try to explain in a comment what this regex is doing (what it's for --- when does this situation occur)

For the next dev I won't know what this is for and likely not even what a nested address is.

Reply:
Let me know if the comments I added sufficiently provide enough information. :)

Reply:
yup that works.